code/modules/projectiles/guns/ballistic/revolver.dm

```diff
@@ -74,7 +75,7 @@
 		. += "It can be spun with <b>alt+click</b>"
 
 /obj/item/gun/ballistic/revolver/detective
-	name = "\improper Colt Detective Special"
+	name = "\improper Hunters Pride Detective Special"
```

Comment:
Grammar edit, this and every other instance of that manufacturer name should be "Hunter's Pride"